typescript/web/src/components/export-button/export-modal/index.tsx

```diff
@@ -104,8 +104,10 @@ const exportDataset = async ({
   const blobDataset = await (await fetch(exportDatasetUrl)).blob();
   const url = window.URL.createObjectURL(blobDataset);
   const element = document.createElement("a");
+  const extension =
+    format === ExportFormat.Yolo || options.coco?.exportImages ? "zip" : "json";
   element.href = url;
-  element.download = datasetName;
+  element.download = `${datasetName}.${extension}`;
```

Comment:
Was it because the extension was missing in the file name?
Have you made sure it didn't broke anything with local? _(this will be useful for #741 since we need export to work)_

Reply:
Extension was missing in the filename, but in local the browser automatically added the extension based on the mime type of the data, however on workspaces we only ever get a generic type `application/octet-stream`.

Reply:
Ah I see. It's strange that web-browsers have a different behavior depending on whether the file is local of not... Thanks for the explanation.
The solution seems OK then. Nice catch 👍

Reply:
> Ah I see. It's strange that web-browsers have a different behavior depending on whether the file is local of not... Thanks for the explanation. The solution seems OK then. Nice catch 👍

It's not depending on whether the file is local or not, it's depending on whether the file is stored in Dexie or in S3 :)